src/main/java/com/payline/payment/equens/utils/PluginUtils.java

```diff
@@ -139,4 +139,30 @@ public static String filter(List<Aspsp> aspsps, String bic) {
         return goodAspsps.get(0).getAspspId();
     }
 
+    /**
+     * put X for every characters in the IBAN except the 4 first, the 4 (or 5) last and " "
+     *
+     * @param iban the String to hide
+     * @return String hided
+     */
+    public static String hideIban(String iban) {
```

Comment:
Algo à revoir je pense,
Il doit y avoir moyen de faire plus simple avec des substring et des replaces avec pattern pour gérer ces cas. La déclaration de la boucle for me parait compliqué.

Reply:
Le probleme c'est que les IBAN ne sont pas les memes pour tous les Pays
mais j'ai rendu le code plus lisible :)